src/plugins/score-plugin-remotecontrol/RemoteControl/HttpServer.cpp

```diff
@@ -283,13 +283,17 @@ Http_server::start_thread()
 
 // Open a server using sockets
 int
-Http_server::open_server()
+HttpServer::open_server()
 {
     try
     {
         auto const address2 = net::ip::make_address("0.0.0.0");
         auto const port = static_cast<unsigned short>(std::atoi("8080"));
-        auto const m_docRoot = std::make_shared<std::string>("./src/plugins/score-plugin-remotecontrol/CMakeFiles/score_plugin_remotecontrol.dir/RemoteControl/build-wasm/");
+        // auto const m_docRoot = std::make_shared<std::string>("./src/plugins/score-plugin-remotecontrol/CMakeFiles/score_plugin_remotecontrol.dir/RemoteControl/build-wasm/");
+        std::string packagesPath = score::AppContext().settings<Library::Settings::Model>().getPackagesPath().toUtf8().constData();
+        QDebug << "packagesPath :" << packagesPath;
```

Comment:
No need for the QDebug anymore (.. does that compile ? it's supposed to be qDebug())